modules/images/webp-uploads/load.php

```diff
@@ -8,6 +8,13 @@
  * @package performance-lab
  */
 
+// Define the constant.
+if ( defined( 'WEBP_UPLOADS_VERSION' ) ) {
+	return;
+}
+
+define( 'WEBP_UPLOADS_VERSION', 'Performance Lab ' . PERFLAB_VERSION );
```

Comment:
This will need to be modified as part of the build step, correct? Do we already have this covered in an issue somewhere?

Reply:
It's part of this PR :)

https://github.com/WordPress/performance/pull/670/files#diff-69937e306961cbecc9a03ad8e84f06ee230caeb86a72a6f222f071df3a518bc3R88

Reply:
🤦🏻‍♂️ of course it is. Thanks for pointing out.